### ISSUE TITLE
Issues in "Note on Persistence" in DID Syntax section

### ISSUE THREAD

**Issue by shigeya** [w3c/did-core#504]
Regarding the 3.1 DID Syntax Section, there is a note on persistence. 
I observe two issues.

- Terminology `UUID` is mentioned only once in the entire document. Do we need to have the UUID terminology entry just for this description?
- I think this note does not belong to the "DID Syntax" section

I propose eliminating UUID terminology entry and adding a reference to UUID since mentioning UUID itself seems useful.

Where to relocate is a question. I'm currently re-reviewing whole "Notes" and related paragraphs; I observe there are some paragraphs we want to transform into Notes, and the others require relocation. I'll post another issue to discuss how notes need to be reorganized after the review.

**Comment by kdenhartog** [comment]
Related to this work is currently being revised as well by @talltree in https://github.com/w3c/did-core/pull/457

I think we can incorporate this feedback into that PR and close once #457 is merged.

**Comment by shigeya** [comment]
@kdenhartog I see. I will check #457

**Comment by shigeya** [comment]
@kdenhartog Read thorough #457.  Looks like it already became beyond just a note and it needs relocation. I will comment in #457.

**Comment by msporny** [comment]
This issue will be closed once PR #457 is merged.

**Comment by talltree** [comment]
Per @shigeya's suggestion, as well as the feedback in the last DID WG special topic call, I revised #457 to move it to the Security Considerations section.

**Comment by shigeya** [comment]
As PR #457 and follow-up PR  #574 discussing persistence and close-to-be-approved PR #574 relocate the text to the security section, this issue can be closed once PR #574 merges.

**Comment by msporny** [comment]
PR #574 has been merged, closing.
